h2o-extensions/target-encoder/src/main/java/ai/h2o/targetencoding/TargetEncoderModel.java

```diff
@@ -119,7 +123,7 @@ private TwoDimTable constructSummary(){
    */
   public Frame transform(Frame data, byte strategy, double noiseLevel, final boolean useBlending, BlendingParams blendingParams,
```

Comment:
off-topic: no plan have an alternative signature using the corresponding enum instead of `byte` for the `strategy` param?
Looks like this signature is for backwards compatibility, but it's not very user-friendly.

Reply:
@seb-h2o I agree that this might be improved but this is out of scope of this PR. Maybe you can create jira which could become a simple `good first task`.